common/config/rush/pnpm-lock.yaml

```diff
@@ -9281,6 +9281,10 @@ packages:
     engines: {node: '>=4.0'}
     dev: false
 
+  /estree-walker/0.6.1:
```

Comment:
I believe this file should not have changed, since you didn't update anything in any `package.json`.
Could you revert it?

Reply:
> Thank you for the pull request! :) And congrats on your first pull request! 🎉 👯‍♂️
> 
> The component looks good but the `border-color` makes it weird when zooming. Also I requested some additional stories.

Thanks for the detailed review! I'll work on the requested changes.

Reply:
I think I have reverted it.